demisto_sdk/commands/validate/validators/IF_validators/IF119_select_values_cannot_contain_multiple_or_only_empty_values_in_single_select_types.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import GitStatuses, IncidentFieldType
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = IncidentField
+
+
+def select_values_do_not_contain_multiple_or_only_empty_values_in_single_select_types(
+    content_item: ContentTypes,
+) -> bool:
+    if content_item.data.get("type") == IncidentFieldType.SINGLE_SELECT:
```

Comment:
Let's add type as an attr of the IncidentField class.

Reply:
There is already a field_type attr